nbnet.h

```diff
@@ -3218,7 +3215,7 @@ void NBN_ConnectionRequestMessage_Destroy(NBN_ConnectionRequestMessage *msg)
 
 int NBN_ConnectionRequestMessage_Serialize(NBN_ConnectionRequestMessage *msg, NBN_Stream *stream)
 {
-    NBN_SerializeUInt(stream, msg->length, 0, NBN_CONNECTION_DATA_MAX_SIZE);
+    NBN_SerializeUInt(stream, msg->length, 1, NBN_CONNECTION_DATA_MAX_SIZE);
```

Comment:
what's the reason behind this change?

it seems that it fails the pipeline:

`client: /home/runner/work/nbnet/nbnet/soak/../nbnet.h:3218: NBN_ConnectionRequestMessage_Serialize: Assertion `msg->length >= 1 && msg->length <= 512' failed.`

it's possible to not send any data upon connection, in this case the length is 0.